src/_P073_7DGT.ino

```diff
@@ -299,26 +466,158 @@ static const byte CharTableMAX7219[42] = {
+//   - pos 40    - uppercase X "X"                      B00110111
+//   - pos 41-66 - Letters from A to Z Siekoo style
+// static const byte SiekooTableTM1637[67] = {
+//   B00111111, B00000110, B01011011, B01001111, B01100110, B01101101,
+//   B01111101, B00000111, B01111111, B01101111, B00000000, B01000000,
+//   B01100011, B01001000, B01010010, B00001000, B00100100, B00010111,
+//   B00010000, B00001100, B00001010, B00001001, B01000110, B01001001,
+//   B00110110, B01101011, B01001011, B00100000, B00100010, B00100001,
+//   B00000011, B01100100, B00111001, B00001111, B00000001, B00111001,
+//   B00110111, B00111111, B00110011, B00111110, B01110110, B01011111, /* A */
+//   B01111100, B01011000, B01011110, B01111001, B01110001, B00111101,
+//   B01110100, B00010001, B00001101, B01110101, B00111000, B01010101,
+//   B01010100, B01011100, B01110011, B01100111, B01010000, B00101101,
+//   B01111000, B00011100, B00101010, B01101010, B00010100, B01101110,
+//   B00011011 };
+static const byte SiekooTableMAX7219[67] = {
+  B01111110, B00110000, B01101101, B01111001, B00110011, B01011011,
+  B01011111, B01110000, B01111111, B01111011, B00000000, B00000001,
+  B01100011, B00001001, B00100101, B00001000, B00010010, B01110100,
+  B00000100, B00011000, B00101000, B01001000, B00110001, B01001001,
+  B00110110, B01101011, B01101001, B00000010, B00100010, B01000010,
+  B01100000, B00010011, B01001110, B01111000, B01000000, B01001110,
+  B01110110, B01111110, B01100110, B00111110, B00110111, B01111101, /* A */
+  B00011111, B00001101, B00111101, B01001111, B01000111, B01011110,
+  B00010111, B01000100, B01011000, B01010111, B00001110, B01010101,
+  B00010101, B00011101, B01100111, B01110011, B00000101, B01011010,
+  B00001111, B00011100, B00101010, B00101011, B00010100, B00111011,
+  B01101100 };
+
+// dSEG7 https://www.keshikan.net/fonts-e.html
+// as the 'over score' character isn't normally available, the pipe "|" is used for that, and for degree the "^"" is used
+// specials:
+//   - pos 0-9   - Numbers from 0 to 9
+//   - pos 10    - Space " "
+//   - pos 11    - minus symbol "-"
+//   - pos 12    - degree symbol "°" (specially handled "^" into a degree)
+//   - pos 13    - equal "="
+//   - pos 14    - slash "/"                            
+//   - pos 15    - underscore "_"
+//   - pos 16-41 - Letters from A to Z dSEG7 style
+// static const byte Dseg7TableTM1637[42] = {
+//   B00111111, B00000110, B01011011, B01001111, B01100110, B01101101,
+//   B01111101, B00000111, B01111111, B01101111, B00000000, B01000000,
+//   B01100011, B01001000, B01001001, B00001000, B01110111, B01111100, /* AB */
+//   B01011000, B01011110, B01111001, B01110001, B00111101, B01110100,
+//   B00000100, B00011110, B01110101, B00111000, B00110111, B01010100,
+//   B01011100, B01110011, B01100111, B01010000, B01101100, B01111000,
+//   B00011100, B00111110, B01111110, B01110110, B01101110, B00011011 };
+static const byte Dseg7TableMAX7219[42] = {
```

Comment:
Why not using PROGMEM for these?
Make sure you address them using _P functions if you access them.

Reply:
That's unknown territory to me, can you provide an example, please?

Reply:
I just took the plunge; a bit of reading on 'Arduino PROGMEM' and a peek in P036 for an example, a few minor changes in the code, and voila, it's running fine 😄 👍